src/main/scala/Mathematics/BinaryExponentiation.scala

```diff
@@ -0,0 +1,22 @@
+package Mathematics
+
+object BinaryExponentiation {
+	/**
+	    * Method returns the binary exponentiation of a given number 
+	      when base and power are passed the parameters
+	    *
+	    * @param Int, Int
+    	    * @return
+    	*/
+	
+	def BinaryExponentiation(base : Int, power : Int): Int = {
```

Comment:
method name should start with lower case to be consistent with the rest of your implementations.

Reply:
done